src/main/java/org/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -597,12 +601,13 @@ public void guiActionInsertEntry(List<BibEntry> entries,
         } catch (DisposedException ex) {
             OOError.from(ex).setTitle(errorTitle).showErrorDialog(dialogService);
         } catch (CreationException
-                | IllegalTypeException
-                | NotRemoveableException
-                | PropertyVetoException
-                | WrappedTargetException ex) {
+                 |
+                 WrappedTargetException ex) {
             LOGGER.warn("Could not insert entry", ex);
             OOError.fromMisc(ex).setTitle(errorTitle).showErrorDialog(dialogService);
+        } catch (
+                IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Do not throw a runtime exception, this will break the whole jabref,.
eitehr show a dialog in gui or logger.error(....)